packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -210,6 +224,18 @@ export class OdspDelayLoadedDeltaStream {
 		});
 	}
 
+	private readonly signalHandler = (signalsArg: ISignalMessage | ISignalMessage[]) => {
+		const signals = Array.isArray(signalsArg) ? signalsArg : [signalsArg];
+		signals.forEach((signal: ISignalMessage) => {
+			const envelope = JSON.parse(signal.content as string) as ISignalEnvelope;
+			if (envelope.contents.type === "PolicyLabelsUpdate") {
+				this.emitDeltaConnectionUpdateEvent({
+					sensitivityLabelsInfo: envelope.contents.content,
```

Comment:
BTW, same issue as above with signals potentially coming before client had a chance to subscribe for events.
In might be Ok for other signals (as we accumulate them), but here we are "processing" signals, and thus we can't process them later (in other loader layers as it's done here).

Reply:
Moved event to service and also processed initial signals.